apps/admin-portal/src/components/identity-providers/settings/outbound-provisioning-settings.tsx

```diff
@@ -115,76 +205,224 @@ export const OutboundProvisioningSettings: FunctionComponent<ProvisioningSetting
+                                                            },
+                                                        ],
+                                                        content: (
+                                                            <OutboundProvisioningConnectorFormFactory
+                                                                metadata={ connector.meta }
+                                                                initialValues={ connector.data }
+                                                                onSubmit={ handleConnectorConfigFormSubmit }
+                                                                type={ connector.meta?.name }
+                                                            />
+                                                        ),
+                                                        id: connector?.id,
+                                                        title: connector?.meta?.displayName,
+                                                    }
+                                                })
+                                            }
+                                        />
+                                    </Grid.Column>
+                                </Grid.Row>
+                            </Grid>
+                        </div>
+                    )
+                    : <ContentLoader/>
+                ) : (
+                    <Grid>
+                        <Grid.Row>
+                            <Grid.Column width={ 8 }>
+                                <Segment>
+                                    <EmptyPlaceholder
+                                        title="No outbound provisioning connectors"
+                                        image={ EmptyPlaceholderIllustrations.emptyList }
+                                        subtitle={ [ "This IDP has no outbound provisioning connectors configured",
+                                            "Add a connect to view it here." ] }
+                                        imageSize="tiny"
+                                        action={ (
+                                            <PrimaryButton onClick={ () => setShowWizard(true) }>
+                                                <Icon name="add"/>
+                                                New Connector
+                                            </PrimaryButton>
+                                        ) }
                                     />
-                                }
-                                <Divider hidden/>
-                            </>
-                        )}
-                </div>
-            )
-            : <ContentLoader/>
+                                </Segment>
+                            </Grid.Column>
+                        </Grid.Row>
+                    </Grid>
+                )
+            }
+            {
+                deletingConnector && (
```

Comment:
if we use "deletingConnector" when we try to delete two provisioning connectors consecutively, won't this code rendered in the page

Reply:
Since we update the identity provider after the delete it won't be an issue.